Program.cs

```diff
@@ -1,4 +1,55 @@
-﻿using System;
-Console.WriteLine ("Hello, World!");
+﻿// --------------------------------------------------------------------------------------------
+// Training ~ A training program for interns at Metamation, Batch - July 2023
+// Copyright (c) Metamation India.                                              
+// ------------------------------------------------------------------------
+// Program.cs
+// CHOCOLATE WRAPPERS 
+// Write a method, for a given money X along with price P of a chocolate and required wrappers W for a chocolate in exchange, return the maximum number of chocolates C you can get along with any unused money X and wrappers W. 
+// For example,Input:X = 15, P = 4, W = 3 Output:C = 4, X = 3, W = 1
 
-   
\ No newline at end of file
+using System;
+
+namespace Training {
+   #region Program ------------------------------------------------------------------------------
+   /// <summary>Chocolate Wrappers</summary>
+   internal class Program {
+      #region Methods ---------------------------------------------
+      /// <summary>Getting input from the user</summary>
+      /// <param name="args">arguments</param>
+      static void Main (string[] args) {
+         Console.Write ("Enter the amount :"); // Amount user having
+         int.TryParse (Console.ReadLine (), out int x);
+         Console.Write ("Enter the price of a chocolate :"); // cost of a chocolate
+         int.TryParse (Console.ReadLine (), out int p);

```

Comment:
Have a function to get valid integer input and use it to the get the value. Keep the prompt going until the user enters a proper value.